README.md

```diff
@@ -145,133 +145,66 @@ You can send metrics, histograms, or trace data from your application to the Wav
 * Flush interval - Interval for flushing data from the `WavefrontSender` directly to Wavefront.
 * Batch size - Amount of data to send to Wavefront in each flush interval.
 
 Together, the batch size and flush interval control the maximum theoretical throughput of the `WavefrontSender`. Override the defaults _only_ to set higher values.
 
-```java
-// Create a builder with the URL of the form "https://DOMAIN.wavefront.com"
-// and a Wavefront API token with direct ingestion permission
-WavefrontDirectIngestionClient.Builder wfDirectIngestionClientBuilder =
-  new WavefrontDirectIngestionClient.Builder(wavefrontURL, token);
-
-// Optional configuration properties.
-// Only override the defaults to set higher values.
-
-// This is the size of internal buffer beyond which data is dropped
-// Optional: Set this to override the default max queue size of 50,000
-wfDirectIngestionClientBuilder.maxQueueSize(100_000);
-
-// This is the max batch of data sent per flush interval
-// Optional: Set this to override the default batch size of 10,000
-wfDirectIngestionClientBuilder.batchSize(20_000);
-
-// Together with batch size controls the max theoretical throughput of the sender
-// Optional: Set this to override the default flush interval value of 1 second
-wfDirectIngestionClientBuilder.flushIntervalSeconds(2);
+**Example**: Use a factory class to create a WavefrontClient and send data to Wavefront via Wavefront Proxy. 
 
-// Finally create a WavefrontDirectIngestionClient
-WavefrontSender wavefrontSender = wfDirectIngestionClientBuilder.build();
- ```
-
-### Option: 3 Sending Data Via A WavefrontClient
-
-Use `WavefrontClientFactory` to create an instance of a `WavefrontClient` which can send either to Proxies or directly to a Wavefront service.
-
-The `WavefrontClientFactory` supports multiple client bindings and will create a `WavefrontMultiClient` for sending to multiple Wavefront services if more than one client configuration is specified.
-
->**Prerequisites**  
-> Before your application can use a WavefrontClient you must either [set up and start a Wavefront proxy](https://docs.wavefront.com/proxies_installing.html).  
-> OR 
-> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups, Roles, and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
-> * The URL of your Wavefront instance. This is the URL you connect to when you log in to Wavefront, typically something like `https://<domain>.wavefront.com`.
-> * [Obtain the API token](http://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
-
-
-#### Initialize the WavefrontClient
-
-You initialize a `WavefrontClient` by building it with the information you obtained in the Prerequisites section.
+```java
+// Add a client with the following URL format: "proxy://<your.proxy.load.balancer>.com"
```

Comment:
Don't we need to specify a port? 
(cc @MikeMcMahon)

Reply:
Is it going to be in this format (proxy 7.0 onwards): http://<Proxy_Host>:2878

Reply:
Is it : `http://<domain>.wavefront.com:2878` now?

Reply:
for proxy ingestion you are correct @basilisk487 , it would be something like `proxy://<your.proxy.load.balancer.com>:<somePort>`

Reply:
> Is it : `http://<domain>.wavefront.com:2878` now?

when sending to wavefront no port is required, that would fall under Direct Data Ingestion and sends over https w/o port specification.